README.md

```diff
@@ -72,6 +72,28 @@ Cookies.remove('name', { path: '/' }); // => true
 
 *Note: when deleting a cookie, you must pass the exact same path, domain and secure options that were used to set the cookie, unless you're relying on the default options that is.*
 
+## JSON
+
+js-cookie provides automatic JSON storage for cookies.
+
+When creating a cookie you can pass an Array or Object Literal instead of a string in the value. If you do so, js-cookie store the string representation of the object according to the `JSON.stringify` api (if available):
```

Comment:
> but Object can be anything, it can be a Date instance, a String instance, a custom object with methods without any relationship with key:value pairs, etc.

Now I get your point. I was specifically remembering that once I needed to fix an issue so this works:

```js
$.cookie("foo", new String("..."))
```

Than I thought, if we say "literals" we might be unnecessarily restricting users to use literals. I didn't think of all the other objects. I just thought `new Array` and `new Object` will probably work just as well...